### PR TITLE
#84202 | Ensure Month Picker in Tribe Bar Respects All Date Formats

### PR DESCRIPTION
**Pull request by ggwicz**
_Ref:_ [C#84202](http://central.tri.be/issues/84202)

### Code Review Note

Many of these changes are just whitespace/tidying-up. The meaningful _code changes_ are located in the `tribe-events.js` file (and its minified equivalent).

### CODE REVIEW

File: src/resources/js/tribe-events-ajax-calendar.js [hunk 1]
```diff
@@ -83,12 +83,11 @@
 
 				ts.mdate = e.date;
 
-				var year = e.date.getFullYear(),
-					month = ('0' + (e.date.getMonth() + 1)).slice( -2 );
+				var year  = e.date.getFullYear(),
+					month = ( '0' + ( e.date.getMonth() + 1 ) ).slice( -2 );
```
Comment (bordoni) [inline src/resources/js/tribe-events-ajax-calendar.js]: Go with `var month`;

File: src/resources/js/tribe-events-ajax-calendar.js [hunk 2]
```diff
@@ -142,15 +143,15 @@
 		}
 
 		function tribe_mobile_setup_day( $date ) {
-			var data = $date.data( 'tribejson' );
+			var data  = $date.data( 'tribejson' );
 			data.date = $date.attr( 'data-day' );
 
-			var $calendar = $date.parents( '.tribe-events-calendar' ),
+			var $calendar  = $date.parents( '.tribe-events-calendar' ),
 				$container = $calendar.next( '#tribe-mobile-container' ),
```
Comment (bordoni) [inline src/resources/js/tribe-events-ajax-calendar.js]: Go with multiple `var name = ...` statements

File: src/resources/js/tribe-events-ajax-calendar.js [hunk 3]
```diff
@@ -171,9 +172,9 @@
 
 		function tribe_mobile_month_setup() {
 
-			var $today = $wrapper.find( '.tribe-events-present' ),
+			var $today          = $wrapper.find( '.tribe-events-present' ),
 				$mobile_trigger = $wrapper.find( '.mobile-trigger' ),
-				$tribe_grid = $wrapper.find( document.getElementById( 'tribe-events-content' ) ).find( '.tribe-events-calendar'  );
+				$tribe_grid     = $wrapper.find( document.getElementById( 'tribe-events-content' ) ).find( '.tribe-events-calendar'  );
```
Comment (bordoni) [inline src/resources/js/tribe-events-ajax-calendar.js]: Go with multiple `var name = ...` statements

File: src/resources/js/tribe-events.js [hunk 4]
```diff
@@ -1293,16 +1306,16 @@ Date.prototype.format = function( mask, utc ) {
...
 		tf.update_viewport_variables();
 
-		var $body = $( 'body' ),
-			$tribe_events = $( '#tribe-events' ),
-			$tribe_content = $( '#tribe-events-content' ),
-			$tribe_events_header = $( '#tribe-events-header' ),
+		var $body                 = $( 'body' ),
+			$tribe_events        = $( document.getElementById( 'tribe-events' ) ),
+			$tribe_content       = $( document.getElementById( 'tribe-events-content' ) ),
+			$tribe_events_header = $( document.getElementById( 'tribe-events-header' ) ),
```
Comment (bordoni) [inline src/resources/js/tribe-events.js]: Go with multiple `var name = ...` statements

**Comment by ggwicz** [comment]
ready for another review 😎

**Pull request merged**